flexmeasures/ui/crud/assets.py

```diff
@@ -303,7 +303,7 @@ def get(self, id: str):
         )
 
     @login_required
-    @route("/<id>/status")
+    @route("/<id>/status/")
```

Comment:
@nhoening no trailing space?

Reply:
Indeed, there should be no slash.
I had [removed it in the first PR](https://github.com/FlexMeasures/flexmeasures/pull/941/commits/03a39f24254c36776e30d434a3512cfdf4551528), but that got mixed up here.

Good catch!